src/libraries/System.IO/tests/Stream/Stream.NullTests.cs

```diff
@@ -194,15 +242,10 @@ public void DisposeAsync_Nop()
             Stream.Null.Write(new byte[42]); // still usable
         }
 
-        public static IEnumerable<object[]> NullReaders
-        {
-            get
-            {
-                yield return new object[] { TextReader.Null };
-                yield return new object[] { StreamReader.Null };
-                yield return new object[] { StringReader.Null };
-            }
-        }
+        public static IEnumerable<object[]> NullReaders =>
+            new[] { TextReader.Null, StreamReader.Null, StringReader.Null }
+                .Distinct()
+                .Select(x => new object[] { x });
```

Comment:
can we revert these changes now?

```suggestion
        public static IEnumerable<object[]> NullReaders
        {
            get
            {
                yield return new object[] { TextReader.Null };
                yield return new object[] { StreamReader.Null };
                yield return new object[] { StringReader.Null };
            }
        }
```